tests/Generators/GoogleGeneratorTest.php

```diff
@@ -35,4 +35,12 @@ public function it_correctly_generates_all_day_events_by_dates(): void
             $this->generator()->generate($this->createEventMultipleDaysViaStartEndWithTimezoneLink())
         );
     }
+
+    /** @test */
+    public function it_can_generate_an_url_with_custom_parameters(): void
+    {
+        $link = $this->createShortEventLink();
+
+        $this->assertMatchesSnapshot($link->google(['recur' => 'RRULE:FREQ=DAILY']));
```

Comment:
Oh google does support re-occuring!
Did you find documentation for the params?

Reply:
My company also maintains docs for services like Google calendar: https://github.com/InteractionDesignFoundation/add-event-to-calendar-docs/blob/master/services/google.md

Links to these docs you can find on PHPDoc (class-level docs on generators)

Reply:
Ahh awesome. Didn't realise google support recurring rules. Shame other providers dont